dockstore-integration-testing/src/test/java/io/dockstore/client/cli/CromwellIT.java

```diff
@@ -165,7 +165,7 @@ public void testRunWorkflow() throws IOException, ApiException {
         File workflowFile = new File(ResourceHelpers.resourceFilePath("hello_world.wdl"));
         File parameterFile = new File(ResourceHelpers.resourceFilePath("hello_world.json"));
         // run a workflow
-        final long run = wdlClient.launch(workflowFile.getAbsolutePath(), true, null, parameterFile.getAbsolutePath(), null, null, null);
+        final long run = wdlClient.launch(workflowFile.getAbsolutePath(), true, null, parameterFile.getAbsolutePath(), null, null, null, null);
```

Comment:
This is getting excessive, might want to create an issue to switch this to a builder pattern. 
This would also help with the big "if" in the launch method and the warning.